src/current/v23.2/manage-a-backup-schedule.md

```diff
@@ -135,7 +135,7 @@ Or nest a [`SELECT` clause]({% link {{ page.version.version }}/select-clause.md
 > DROP SCHEDULES WITH x AS (SHOW SCHEDULES) SELECT id FROM x WHERE label = 'schedule_database';
 ~~~~
 
-For more information, see [`DROP SCHEDULES`]({% link {{ page.version.version }}/drop-schedules.md %}).
+When `DROP SCHEDULES` removes a [scheduled full backup]({% link {{ page.version.version }}/create-schedule-for-backup.md %}#create-a-schedule-for-full-backups-only-core), all associated [scheduled incremental backups]({% link {{ page.version.version }}/create-schedule-for-backup.md %}#incremental-backup-schedules) are dropped. For more information, see [`DROP SCHEDULES`]({% link {{ page.version.version }}/drop-schedules.md %}).
```

Comment:
nit: this block still needs to be changed.

Reply:
fixed